Source/WebCore/platform/audio/cocoa/AudioSampleBufferConverter.mm

```diff
@@ -250,7 +259,29 @@
         UInt32 size = sizeof(primeInfo);
         if (auto error = PAL::AudioConverterGetProperty(m_converter, kAudioConverterPrimeInfo, &size, &primeInfo))
             RELEASE_LOG_ERROR(MediaStream, "AudioSampleBufferConverter getting kAudioConverterPrimeInfo failed with %d", static_cast<int>(error));
+        else
+            m_preSkip = primeInfo.leadingFrames;
         m_remainingPrimeDuration = PAL::CMTimeMake(primeInfo.leadingFrames, m_destinationFormat.mSampleRate);
+
+        if (m_options.bitrateMode) {
+            UInt32 bitrateMode = *m_options.bitrateMode == BitrateMode::Variable ? kAudioCodecBitRateControlMode_Variable : kAudioCodecBitRateControlMode_Constant;
+            if (auto error = PAL::AudioConverterSetProperty(m_converter, kAudioCodecPropertyBitRateControlMode, sizeof(bitrateMode), &bitrateMode))
+                RELEASE_LOG_ERROR(MediaStream, "AudioSampleBufferConverter setting kAudioCodecPropertyBitRateControlMode failed with %d", static_cast<int>(error));
```

Comment:
I would use RELEASE_LOG_ERROR_IF here.
Ditto below

Reply:
hmmm, that _IF is redundant though, we are there because error isn't null. Maybe it's better to remove all the other RELEASE_LOG_ERROR_IF instead

Reply:
and using RELEASE_LOG_ERROR_IF and running the command there, wouldn't let us print the error value in the message.
So I'll leave it as-is.